### ISSUE TITLE
test case setup_vm failed in brand new x86 VM test environment

### ISSUE THREAD

**Issue by hu-weihua** [xcat2/xcat-core#5735]
Test case setup_vm failed in brand new x86 VM test environment.

```
RUN:if [ "x86_64" != "ppc64"  -a  "kvm" != "ipmi" -a "kvm" != "openbmc" ];then echo "CN node c910f04x12v04 is a vm which mgt is kvm, start to recreate the vm now"; echo "rmvm c910f04x12v04 -f -p";rpower c910f04x12v04 off; sleep 3; rmvm c910f04x12v04 -f -p; var=`expr substr "dir:///var/lib/libvirt/images/" 1 3`; echo "The disk create way of c910f04x12v04 is $var"; if [ "$var" = "phy" ]; then echo "mkvm c910f04x12v04"; mkvm c910f04x12v04; exit $?; elif [ "$var" = "dir" ]; then echo "mkvm c910f04x12v04 -s 20G"; mkvm c910f04x12v04 -s 20G; exit $?; elif ["$var" = "nfs" -o "$var" = "lvm" ];then echo  "Need to fix me"; exit 2; else echo "unsupported disk creation way"; exit 3;fi;else echo "CN node c910f04x12v04 is a non-VM; do not need to recreate it";fi [Thu Oct 25 08:16:31 2018]
ElapsedTime:6 sec
RETURN rc = 1
OUTPUT:
CN node c910f04x12v04 is a vm which mgt is kvm, start to recreate the vm now
rmvm c910f04x12v04 -f -p
c910f04x12v04: off
c910f04x12v04: [c910f04x12v02]: Error: Cannot remove guest vm, no such vm found
The disk create way of c910f04x12v04 is dir
mkvm c910f04x12v04 -s 20G
c910f04x12v04: [c910f04x12v02]: Error: Storage creation request conflicts with existing file(s) c910f04x12v04.sda.qcow2. To force remove the existing storage file, rerun mkvm with the -f option.
CHECK:rc == 0	[Failed]
```

Due to in branch new x86 environment, automation main program just input node definition into xcat, so there is no VM xml file in xcat db, in this case, run ``rmvm <node> -f -p`` will trigger error ``Error: Cannot remove guest vm, no such vm found``.

**Comment by hu-weihua** [comment]
Have been fixed by pr #5736

**Comment by hu-weihua** [comment]
The fix still has problem:
```
RUN:if [ "x86_64" != "ppc64"  -a  "kvm" != "ipmi" -a "kvm" != "openbmc" ];then echo "SN node c910f04x12v03 is a vm which mgt is kvm, start to recreate the vm now"; echo "rpower c910f04x12v03 off"; rpower c910f04x12v03 off; sleep 3; echo "rpower c910f04x12v03 stat"; rpower c910f04x12v03 stat; var=`expr substr "dir:///var/lib/libvirt/images/" 1 3`; echo "The disk create way of c910f04x12v03 is $var"; if [ "$var" = "phy" ]; then echo "mkvm c910f04x12v03"; mkvm c910f04x12v03; echo "rmvm c910f04x12v03 -f -p"; rmvm c910f04x12v03 -f -p; echo "mkvm c910f04x12v03"; mkvm c910f04x12v03; exit $?; elif [ "$var" = "dir" ]; then echo "mkvm c910f04x12v03 -s 20G"; mkvm c910f04x12v03 -s 20G; echo "rmvm c910f04x12v03 -f -p"; rmvm c910f04x12v03 -f -p; echo "mkvm c910f04x12v03 -s 20G"; mkvm c910f04x12v03 -s 20G; exit $?; elif ["$var" = "nfs" -o "$var" = "lvm" ];then echo  "Need to fix me"; exit 2; else echo "unsupported disk creation way"; exit 3;fi;else echo "SN node c910f04x12v03 is a non-VM; do not need to recreate it";fi [Tue Oct 30 11:20:34 2018]
ElapsedTime:7 sec
RETURN rc = 1
OUTPUT:
SN node c910f04x12v03 is a vm which mgt is kvm, start to recreate the vm now
rpower c910f04x12v03 off
c910f04x12v03: off
rpower c910f04x12v03 stat
c910f04x12v03: off
The disk create way of c910f04x12v03 is dir
mkvm c910f04x12v03 -s 20G
c910f04x12v03: [c910f04x12v02]: Error: Storage creation request conflicts with existing file(s) c910f04x12v03.sda.qcow2. To force remove the existing storage file, rerun mkvm with the -f option.
rmvm c910f04x12v03 -f -p
c910f04x12v03: [c910f04x12v02]: Error: Cannot remove guest vm, no such vm found
mkvm c910f04x12v03 -s 20G
c910f04x12v03: [c910f04x12v02]: Error: Storage creation request conflicts with existing file(s) c910f04x12v03.sda.qcow2. To force remove the existing storage file, rerun mkvm with the -f option.
CHECK:rc == 0   [Failed]
```
